test/Mono.Linker.Tests.Cases/Attributes.Debugger/DebuggerAttributesRemoved.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<linker>
+<linker xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://www.w3schools.com ../../../src/ILLink.Shared/ILLink.LinkAttributes.xsd">
```

Comment:
nit: I'm not experienced on XML schemas, but having a link to w3schools seems like something we don't want